db/migrate/20210225013639_add_status_to_merchants.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToMerchants < ActiveRecord::Migration[5.2]
+  def change
+    add_column :merchants, :status, :integer, default: 1
+  end
```

Comment:
Merchants are created as "enabled" by default (for my own ref while reviewing)

Reply:
Yup!